app/controllers/v1/submissions_controller.rb

```diff
@@ -28,7 +28,7 @@ def create
   def update
     submission = Submission.find(params[:id])
     submission.levy = update_submission_params[:levy] * 100
-    submission.aasm.current_state = 'completed' if update_submission_params[:levy]
+    submission.ready_for_review! if update_submission_params[:levy]
```

Comment:
I have a feeling the `if` isn't needed here - if `levy` is nil, I think the previous `update_submission_params[:levy] * 100` will break before it ever gets here.

Reply:
Good point; updated!